sdk/communication/Azure.Communication.Chat/src/Azure.Communication.Chat.csproj

```diff
@@ -7,9 +7,7 @@
       Microsoft Azure Communication Chat quickstart - https://docs.microsoft.com/azure/communication-services/quickstarts/chat/get-started?pivots=programming-language-csharp
     </Description>
     <AssemblyTitle>Azure Communication Chat Service</AssemblyTitle>
-    <Version>1.1.0-beta.1</Version>
-    <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>1.0.0</ApiCompatVersion>
```

Comment:
The ApiCompatVersion should stay and it is automatic so we don't need to change anything there